### PR TITLE
chore: update taskDefinitions schema

### PR DESCRIPTION
**Pull request by kuojianlu**
- remove debug-npm-install, debug-set-up-tab, debug-set-up-bot, debug-set-up-sso, debug-prepare-manifest
- add provision, deploy
- remove devCert, func, ngrok, dotnet from prerequisites
- add env, output to tunnel

### CODE REVIEW

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -362,40 +362,23 @@
...
     "teamstoolkit.webview.accountHelp": "Account Help",
-    "teamstoolkit.taskDefinitions.command.prerequisites.description": "Validate and install prerequisites.\n See https://aka.ms/teamsfx-check-prerequisites-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.tab.description": "Set up tab.\n See https://aka.ms/teamsfx-debug-set-up-tab-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.bot.description": "Set up bot.\n See https://aka.ms/teamsfx-debug-set-up-bot-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.sso.description": "Set up SSO.\n See https://aka.ms/teamsfx-debug-set-up-sso-task for how to customize following args.",
-    "teamstoolkit.taskDefinitions.command.manifest.description": "Build and upload Teams manifest.\n See https://aka.ms/teamsfx-debug-prepare-manifest-task for how to customize following args.",
+    "teamstoolkit.taskDefinitions.command.prerequisites.description": "Validate prerequisites.\n See https://aka.ms/teamsfx-tasks/check-prerequisites for how to customize following args.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:365]: following -> the following
args -> arguments

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -362,40 +362,23 @@
...
+    "teamstoolkit.taskDefinitions.command.startLocalTunnel.description": "Start the local tunnel service.\n See https://aka.ms/teamsfx-tasks/local-tunnel for how to customize following args.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:366]: tunnel -> tunneling
following args -> the following arguments

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -362,40 +362,23 @@
...
+    "teamstoolkit.taskDefinitions.command.provision.description": "Execute provision lifecycle.\n See https://aka.ms/teamsfx-tasks/provision to know the details and how to customize the args.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:367]: to know the details -> for details
 args -> arguments

File: packages/vscode-extension/package.nls.json [hunk 4]
```diff
@@ -362,40 +362,23 @@
...
+    "teamstoolkit.taskDefinitions.command.deploy.description": "Execute deploy lifecycle.\n See https://aka.ms/teamsfx-tasks/deploy to know the details and how to customize the args.",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:368]: to know the details -> for details
args -> arguments

**Review by swatDong — APPROVED** [review]

File: packages/vscode-extension/package.nls.json (hunk 1 above)
Comment (kuojianlu) [reply packages/vscode-extension/package.nls.json]: Thank you. Updated the wording.

**Review by sffamily — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**